core/shared/src/main/scala/scalaz/zio/Fiber.scala

```diff
@@ -179,6 +179,8 @@ trait Fiber[+E, +A] { self =>
           .fork
     }.flatten
 
+  final def managed: Managed[Any, Nothing, Fiber[E, A]] =
```

Comment:
Isn't the convention to use `from`/`to` prefixes when we convert between types?

Reply:
Hmm, I think there's precedence here with `ZIO#managed`. But we can just change them both; makes them more discoverable.

Reply:
Renamed both `Fiber#managed` and `ZIO#managed` to `toManaged`.